recipes/gmp/all/conanfile.py

```diff
@@ -53,8 +58,10 @@ def _configure_autotools(self):
                 configure_stats = os.stat(configure_file)
                 os.chmod(configure_file, configure_stats.st_mode | stat.S_IEXEC)
             configure_args = []
-            if self.options.disable_assembly:
+            if self.options.get_safe("disable_assembly", False):
                 configure_args.append("--disable-assembly")
+            if self.options.enable_fat:
```

Comment:
also deleted https://github.com/conan-io/conan-center-index/pull/3853/files#diff-83911a377d766eaee32c0ca5293b332c6d242b8f2032797a9871771d9c94f461R27

```suggestion
            if self.options.get_safe("enable_fat", False):
```

Reply:
Right, thanks!